apps/site/src/app/admin/applicants/components/ApplicantStatus.tsx

```diff
@@ -11,6 +11,8 @@ export const StatusLabels = {
 	[Status.pending]: "needs review",
 	[Status.reviewed]: "reviewed",
 	[Status.released]: "released",
+	[Status.signed]: "Waiver Signed",
+	[Status.confirmed]: "Confirmed",
```

Comment:
My mistake, I was under the impression that all of the statuses were title cased. Please convert these back to lowercase the way you had them before.